scripts/ebs-warmer.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Source: https://gist.github.com/muhqu/3293988
+# Ref: https://forums.aws.amazon.com/message.jspa?messageID=198413
+# Ref: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-initialize.html
+
+if [ -z "$1" ]; then
+    echo "Usage: sudo $0 /dev/sdh1"
+    exit 1;
+fi
+
+dd if=$1 of=/dev/null bs=1M & pid=$!
```

Comment:
This appears to be making progress at ~3.5MB/s on a fresh EBS volume. At that rate, it might be better to try and identify the subset of files that RWD uses and `cat` those to `/dev/null`. That'll force just those blocks to be warm.

Matt's comments on which files to target:

> That seems like it might be preferable.  There are a few large files that exist in `Main_Watershed` that all RWD requests are going to hit, then the branch out into a selection of several 10,000s directories within `Subwatershed_All` depending on the geographic point.  The contents of those files are relatively small, and can probably tolerate not being warmed.  In aggregate their a lot of data, but in practice I don't think an individual call hits many of them.